### PR TITLE
[studio-ui][3.1.x] Shared content data source does not pass default type or path to search when selecting via search #4025

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/4025

### CODE REVIEW

File: static-assets/components/cstudio-search/search.js [hunk 1]
```diff
@@ -620,6 +644,19 @@
     $('#' + sortOrderValue).prop('checked', true);
     $('select[name="sortOrder"]').val(sortOrderValue);
 
+    if (searchContext.path) {
+      $('#filterPath').val(searchContext.path.replace('.+', ''))/*.prop('disabled', true)*/;
```
Comment (rart) [inline static-assets/components/cstudio-search/search.js]: x comment

File: templates/web/search.ftl [hunk 2]
```diff
@@ -124,6 +124,25 @@
                                 </div>
                             </div>
 
+                            <div class="panel panel-default">
+                              <div class="panel-heading" role="tab" id="headingPath">
+                                <h4 class="panel-title">
+                                  <a class="path-header-label" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapsePath"
+                                     aria-expanded="false" aria-controls="collapsePath">
+                                    <span data-trans="path">Path</span>
```
Comment (rart) [inline templates/web/search.ftl:132]: i18n

File: templates/web/search.ftl [hunk 3]
```diff
@@ -124,6 +124,25 @@
...
+                                    <i class="fa fa-check-circle selected hide ml5" aria-hidden="true"></i>
+                                  </a>
+                                </h4>
+                              </div>
+                              <div id="collapsePath" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingPath">
+                                <div class="panel-body">
+                                  <li class="filter-item" filter-name="path">
+                                    <input class="filter-path" id="filterPath" type="text" name="path">
+                                    <button type="button" class="btn btn-primary apply-path mt5" data-trans="apply">Apply</button>
```
Comment (rart) [inline templates/web/search.ftl:141]: i18n

File: templates/web/search.ftl [hunk 4]
```diff
@@ -124,6 +124,25 @@
                                 </div>
                             </div>
 
+                            <div class="panel panel-default">
+                              <div class="panel-heading" role="tab" id="headingPath">
```
Comment (rart) [inline templates/web/search.ftl:128]: is this id repeated?

**Review by rart — CHANGES_REQUESTED** [review]

File: templates/web/search.ftl (hunk 4 above)
Comment (jvega190) [reply templates/web/search.ftl:128]: no it isn't, is the id used for this new filter

**Review by rart — APPROVED** [review]

**Pull request merged**